torch/testing/_internal/common_mps.py

```diff
@@ -0,0 +1,1029 @@
+import unittest
+from collections.abc import Sequence
+from typing import Optional
+
+import torch
+from torch.testing._internal.common_utils import MACOS_VERSION
+from torch.testing._internal.opinfo.core import DecorateInfo, OpInfo
```

Comment:
Is there anything wrong with doing something like that?
```suggestion
from .common_utils import MACOS_VERSION
from .opinfo.core import DecorateInfo, OpInfo
```